master/internal/experiment.go

```diff
@@ -356,8 +356,12 @@ func (e *internalExperiment) TrialReportProgress(msg experiment.TrialReportProgr
 	e.mu.Lock()
 	defer e.mu.Unlock()
 
-	e.searcher.SetTrialProgress(msg.RequestID, msg.Progress)
-	progress := e.searcher.Progress()
+	progress := float64(msg.Progress)
+	if !msg.IsRaw {
+		e.searcher.SetTrialProgress(msg.RequestID, msg.Progress)
```

Comment:
@azhou-determined  Should we only `SetTrialProgress` when the progress is not "raw"?